src/layouts/header.scss

```diff
@@ -197,6 +197,7 @@ $lg: "screen and (max-width : 1024px)";
         text-decoration: none;
         font-weight: 600;
         font-size: 14px;
+        text-align: right;
```

Comment:
This has no impact on the mobile headers as far as I can see (over written by your `text-align: center;` below). I suspect this is also an unintentional revert of my previous PR. Please double-check.